lib/orchestrator/application.rb

```diff
@@ -2,38 +2,34 @@ module Orchestrator
   class Application
 
     def initialize
-      @__queue__ = Concurrent::MVar.new(Queue.new)
-
-      # TOOD: Add a 1sec timer to unlock any locked jobs
-      # that are 15s or older
-
-      # TODO: Remove this temporary thing
-      #enqueue_job!(Job.new("test_runner", 1, 'ruby', 'bob', "s3://exercism-iterations/production/iterations/1182520"))
+      @__client__ = Concurrent::MVar.new(
+        Aws::DynamoDB::Client.new(
+          endpoint: Orchestrator.config.dynamodb_endpoint,
+          profile: "exercism_website",
```

Comment:
The tooling invoker uses the `exercism_tooling_invoker` profile. Should we change that to `exercism_website`, or this to `exercism_tooling_orchestrator`?

Reply:
This is all getting unified everywhere now, hopefully. Will explain it all when i'm done.